cloudbuild.yaml

```diff
@@ -314,13 +314,13 @@ steps:
           region=${versionsAndRegions[$version]}
           if [ $cloudProduct = generic ]
           then
-            featureWithGate="CustomFasSyncInterval=false&SDKGracefulTermination=false&StateAllocationFilter=false&PlayerAllocationFilter=true&PlayerTracking=true&ResetMetricsOnDelete=false&PodHostname=false&SplitControllerAndExtensions=true&FleetAllocationOverflow=true&Example=true"
+            featureWithGate="CustomFasSyncInterval=false&SDKGracefulTermination=false&StateAllocationFilter=false&PlayerAllocationFilter=true&PlayerTracking=true&ResetMetricsOnDelete=false&PodHostname=false&SplitControllerAndExtensions=false&FleetAllocationOverflow=true&Example=true"
```

Comment:
The `featureWithGate` and `featureWithoutGate` variables are now the same between generic and autopilot, so we should pull them out of the if statement and just have them set in one place. This will make it easier for future feature promotions and also help enforce consistency between generic and autopilot installations.

Reply:
I'm going to merge this so we can get it in before the release (since consolidating these lines doesn't have much effect other than a bit of cleanup). Can you send a follow up PR to do this cleanup?

Reply:
I will do that.